.github/workflows/lint-test.yaml

```diff
@@ -39,16 +39,16 @@ jobs:
 
       - name: Run chart-testing (lint)
         id: lint
-        uses: helm/chart-testing-action@v1.0.0-alpha.3 # fixed on v1.0.0-alpha.3 for helm2
+        uses: helm/chart-testing-action@v1.0.0
```

Comment:
This is intended to use `v1.0.0-alpha.3` version to test backward-compatible with helm2.
So, the version should not be bumped. [1]

Reply:
humm, ok, thanks!

Reply:
done @jenting